yarn.lock

```diff
@@ -16,6 +16,13 @@
     esutils "^2.0.2"
     js-tokens "^3.0.0"
 
+"@babel/runtime@^7.0.0-beta.49":
```

Comment:
It appears that `semantic-ui-react` requires Babel at runtime (!!!). I’m
still trying to dig into exactly how they use it, but this is deeply
disturbing. Depending on what they’re doing, it could cripple the
performance of our application. I’ll get back to you on this.